src/codecs/html/html.test.ts

```diff
@@ -1,16 +1,17 @@
 import stencila from '@stencila/schema'
 import fs from 'fs'
 import { dump, load } from '../../util/vfile'
-import { decode, encode } from './'
-import { EncodeOptions } from '../..'
+import { HTML } from './'
+
+const { encode, decode } = new HTML()
```

Comment:
I know @nokome and I are kind of on the opposite ends of the code-naming-verbosity spectrum but having a class just called `HTML` kind of feels like it could get confusing. Not so much with the other codecs but something dealing with HTML is quite common in the Javascript world. Just pointing this out, I don't mind if we keep or change it.

Reply:
I'd be happy to add `Codec` to all the class names e.g. `HtmlCodec` (actually, I think I prefer that :)